frontend/pages/Parts/[deviceHandle]/[itemTypeHandle].tsx

```diff
@@ -0,0 +1,129 @@
+import {
```

Comment:
I'm doubtful that this whole file should be getting added; did it slip in by accident?